docs/source/kapsel/index.rst

```diff
@@ -182,7 +182,7 @@ The command line session will look something like::
 
 If you look at ``kapsel.yml`` now, you'll see bokeh and pandas
 listed under the ``packages:`` section. You will also see files
-such as ``envs/default/bin/bokeh`` since the packages have now
+such as ``envs/default/conda-meta/bokeh`` since the packages have now
```

Comment:
We could also have separate Windows and OS X/Linux instructions. Let me
know...

_______________________________

Kerry Watson
Documentation manager

[image: Continuum Analytics] <http://continuum.io>

On Thu, Aug 4, 2016 at 10:14 AM, Havoc Pennington <notifications@github.com>
wrote:

> In docs/source/kapsel/index.rst
> <https://github.com/conda/conda-docs/pull/329#discussion_r73541710>:
>
> > @@ -182,7 +182,7 @@ The command line session will look something like::
> >
> >  If you look at ``kapsel.yml`` now, you'll see bokeh and pandas
> >  listed under the ``packages:`` section. You will also see files
> > -such as ``envs/default/bin/bokeh`` since the packages have now
> > +such as ``envs/default/conda-meta/bokeh`` since the packages have now
>
> I don't think this is right. I have conda-meta/bokeh-0.12.0-py27_0.json,
> not conda-meta/bokeh.
>
> I suspect the issue is that on Windows Aamna had something like
> Scripts\bokeh.exe, not bin/bokeh.
> I'm having trouble coming up with a cross-platform file to mention, so
> maybe we should just say "you will also see Bokeh-related files in
> envs/default since the packages have now been installed" or maybe we should
> delete this sentence...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda/conda-docs/pull/329/files/295fd7af003ef544c10d189038c66c5edccef29e#r73541710>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEXgDafng-JiLe071g5Ecljq9G1psHkks5qcgHFgaJpZM4JcwIq>
> .
>